File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main (void)
{
    int s = 6;
    for (int i = 0; i < s; i++)
    {
       for (int j = 0; j <s; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: week6/sentimental-mario-less/mario.py
import cs50

def main():
    n = getnumber()
    buildblock(n)









def getnumber():
    while True:
        n = cs50.get_int("What number? ")
        if n > 1:
            return n
        elif n < 8:
            return n

def buildblock(n):
    for x in range(n):
        print(" " * (n-1) + "#" )
        n = n - 1


main()